projects/README.md

```diff
@@ -30,20 +30,22 @@ We also provide some documentation listed below to help you get started:
 
 ## Project List
 
-- [MMPose4AIGC](./mmpose4aigc)
+- **[:zap:RTMPose](./rtmpose)**: Real-Time Multi-Person Pose Estimation toolkit based on MMPose
 
-  This project will demonstrate how to use MMPose to generate skeleton images for pose guided AI image generation.
+  <div align="center">
+  <img src="https://user-images.githubusercontent.com/15977946/225229448-36ff568d-a723-4248-bb19-2df4044ff8e8.png" width=800 height=200 />
+  </div><br/>
 
-  <div align=center>
-  <img src="https://user-images.githubusercontent.com/13503330/222403836-c65ba905-4bdd-4a44-834c-ff8d5959649d.png" width=1000 height=200/>
-  </div>
+- **[:art:MMPose4AIGC](./mmpose4aigc)**: Guide AI image generation with MMPose
 
-- [RTMPose](./rtmpose)
+  <div align=center>
+  <img src="https://user-images.githubusercontent.com/13503330/222403836-c65ba905-4bdd-4a44-834c-ff8d5959649d.png" width="800"/>
+  </div><br/>
 
-  Real-Time Multi-Person Pose Estimation toolkit based on MMPose
+- **[:bulb:YOLOX-Pose](./yolox-pose)**: Enhancing YOLO for Multi Person Pose Estimation Using Object Keypoint Similarity Loss
 
-  <div align="center">
-  <img width=1000 height=200 src="https://user-images.githubusercontent.com/15977946/225229448-36ff568d-a723-4248-bb19-2df4044ff8e8.png"/>
-  </div>
+  <div align=center>
+  <img src="https://user-images.githubusercontent.com/26127467/226550744-3dd948f4-cc5a-4a2f-a737-c595fc6dfe4d.jpg" width="800" style="width: 800px; height: 200px; object-fit: cover"/>
+  </div><br/>
```

Comment:
Perhaps this image would be better suited to the desired size on this page:

<img src="https://user-images.githubusercontent.com/26127467/226655503-3cee746e-6e42-40be-82ae-6e7cae2a4c7e.jpg" width="800" style="width: 800px; height: 200px; object-fit: cover"/>

Reply:
Agreed. Please replace the image here, and in `projects/yolox-pose/README.md`